timescaledb/how-to-guides/migrate-data/different-db.md

```diff
@@ -0,0 +1,75 @@
+# Migrating from a Different PostgreSQL Database
+<highlight type="tip">
+First make sure that you have properly [installed](/timescaledb/latest/how-to-guides/install-timescaledb/)
+**AND [setup](/timescaledb/latest/how-to-guides/install-timescaledb/post-install-setup/)** TimescaleDB
+within your PostgreSQL instance.
+</highlight>
+
+To migrate your database from PostgreSQL to TimescaleDB, you
+need `pg_dump` for exporting your schema and data.
+
+Migration falls into three main steps:
+
+1. Copy over the database schema and choose which tables will become
+hypertables (i.e., those that currently have time-series data).
+1. Backup data to comma-separated values (CSV).
+1. Import the data into TimescaleDB
```

Comment:
```suggestion
1. Copy over the database schema and choose which tables will become
hypertables (i.e., those that currently have time-series data).
2. Backup data to comma-separated values (CSV).
3. Import the data into TimescaleDB
```

Reply:
Doesn't this just create maintenance overhead, and risk numbering errors?

Reply:
Sorry, I didn't verify if gatsby fixes it for us or how the numbers will get rendered. I just thought it was not observed. Is it going to be automatic?

Reply:
It's standard markup, and it does appear to be working properly:

![image](https://user-images.githubusercontent.com/3914967/118230632-56a8ce00-b4d1-11eb-973a-115b5b4de7cd.png)

http://docs-v3-dev.timescale.com.s3-website-us-east-1.amazonaws.com/docs-migrate-links-lana/timescaledb/migrate-links-lana/how-to-guides/migrate-data/different-db/#copying-schema-amp-setting-up-hypertables